### PR TITLE
Feature/cds 13 seg edit

### PR DESCRIPTION
**Pull request by chloehkwong1**
## Description of change

Added dropdown list for two new fields - Export Segment (optional) and Export Sub-segment (optional).

## Test instructions

Cypress tests edited to include new fields in `companies/edit-spec.js` and `companies/business-details-spec.js`. 

### Before
image

### After
image

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -178,8 +178,8 @@
     "@wdio/browserstack-service": "^6.12.1",
     "@wdio/cli": "^6.12.1",
     "@wdio/local-runner": "^7.0.0",
-    "@wdio/reporter": "^7.0.7",
     "@wdio/mocha-framework": "^7.0.7",
+    "@wdio/reporter": "^7.0.7",
```
Comment (rachaelcodes) [inline package.json:182]: I'm a bit concerned about the `package.json` and `package-lock.json` - did these changes appear after you ran the code in Docker or locally?

**Review by rachaelcodes — CHANGES_REQUESTED** [review]
Could you double-check if the changes to the package.json and package-lock.json are needed?

File: package.json (hunk 1 above)
Comment (ccp92) [reply package.json:182]: I think this is the result of a merge conflict we got when we rebased master into our branch (it seems like the order of the packages in `package.json` changed which caused the merge conflict?)
Checking the master branch, 7.0.7 is the correct version.

File: src/apps/companies/apps/edit-company/__test__/transformers.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const { transformCompanyToForm } = require('../transformers')
```
Comment (rachaelcodes) [inline src/apps/companies/apps/edit-company/__test__/transformers.test.js:1]: I've put a comment in Slack about this, and about our new testing framework, in response to Tom's question

Comment (ccp92) [reply src/apps/companies/apps/edit-company/__test__/transformers.test.js]: Thanks @rachaelcodes. We've removed this unit test and will stick to adding the newer functional tests going forwards 😄

File: package.json (hunk 1 above)
Comment (rachaelcodes) [reply package.json:182]: Looks like both versions are 7.0.7, but for some reason the the packages were added out of alphabetical order in the past.

Hm. Let's keep it this way, as it's the right order.

**Review by rachaelcodes — APPROVED** [review]

**Review by tomkingston — APPROVED** [review]

**Review by ian-leggett — APPROVED** [review]

**Pull request merged**